src/network/types.rs

```diff
@@ -173,6 +173,10 @@ pub struct NetInterface {
     /// Subnets list of assigned subnets with their gateway.
     #[serde(rename = "subnets")]
     pub subnets: Option<Vec<NetAddress>>,
+
+    /// Isolation determines whether the network will be able to communicate to others
+    #[serde(rename = "isolation")]
+    pub isolation: bool,
```

Comment:
This should not be returned. Podman already knows this since the setting is in the network config. The return values are only for dynamic values such as ip/mac addresses.